components/Variables/Var.tsx

```diff
@@ -6,24 +6,24 @@ import type { VarsContextProps } from "./context";
 import styles from "./Var.module.css";
 
 interface VarProps {
-  name: string;
+  name?: string;
   description?: string;
   needLabel?: boolean;
-  isGlobal?: boolean;
+  isGlobal?: string;
 }
 
 export const Var = ({
   name,
   description = "",
   needLabel = false,
-  isGlobal = false,
+  isGlobal,
 }: VarProps) => {
   const { fields, setField, addField } =
     useContext<VarsContextProps>(VarsContext);
   const val = fields[name] || "";
 
   useEffect(() => {
-    addField(name, isGlobal, description);
+    addField(name, isGlobal === "true", description);
```

Comment:
should this be a boolean `true` instead of a string `"true"` ?

Reply:
It looked like other places were already calling this with string values and not boolean ones.